Cython/Utility/Optimize.c

```diff
@@ -628,7 +628,7 @@ static double __Pyx__PyObject_AsDouble(PyObject* obj) {
     }
 #endif
     if (likely(float_value)) {
-        double value = PyFloat_AS_DOUBLE(float_value);
+        double value = __pyx_PyFloat_AsDouble(float_value);
```

Comment:
I'd keep the name `_AS_DOUBLE` to hint at what it does "normally". Also because the code annotator (`cython -a`) naively distinguishes between "simple fast macros" and "probably more costly functions" by looking at the letter case.

Reply:
Still TODO, will come back to this

Reply:
Even if `__pyx_PyFloat_AsDouble` already does `PyFloat_AS_DOUBLE `? In that case I'll just add another if directive :)

Reply:
Even if `__pyx_PyFloat_AsDouble ` still does: `PyFloat_AS_DOUBLE ` in a normal compilation mode? Anyways, I brought this back and added an if directive :)